### PR TITLE
Use branched client, if KBC_BRANCHID present

### PR DESCRIPTION
**Pull request by michaljurecko**
Jira: https://keboola.atlassian.net/browse/COM-611

Changes:
- If `KBC_BRANCHID` env var present -> it is used `BranchAwareClient`.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -23,6 +24,12 @@
         "symfony/finder": "^5.1",
         "symfony/process": "^5.0"
     },
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/keboola/sql-formatter"
```
Comment (michaljurecko) [inline composer.json:30]: Tu by sme mali pouzivat nasu kniznicu, ale zabudlo sa to tu pridat - minule.

File: src/ApiClientFactory.php [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+               'token' => $this->config->getStorageApiToken(),
+            ]),
+            null,
+            null
+        );
+
+        $branchId = $this->config->getStorageBranchId();
+        if ($branchId) {
+            $clientWrapper->setBranchId($branchId);
+            return $clientWrapper->getBranchClient();
```
Comment (michaljurecko) [inline src/ApiClientFactory.php:33]: returns `BranchAwareClient` extends `Client`

File: src/ApiClientFactory.php [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        }
+
+        return $clientWrapper->getBasicClient();
```
Comment (michaljurecko) [inline src/ApiClientFactory.php:36]: returns `Client`

**Review by ondrajodas — APPROVED** [review]
👍

**Pull request merged**